CC3D/Models/BiocIU/SARSCoV2MultiscaleVTM/Model/Simulation/CoronavirusSteppables.py

```diff
@@ -646,17 +661,15 @@ def step(self, mcs):
             # print(EC50_ck_immune)
             up_res = self.ck_secretor.uptakeInsideCellTotalCount(cell,
                                                                  max_ck_consume / cell.volume, 0.1)
+            # Added virus uptake
+            self.virus_secretor.uptakeInsideCellTotalCount(cell,max_ck_consume / cell.volume, 0.1)
```

Comment:
we should try to find the uptake rate for viruses. I'm pretty sure the viral concentrations and ck concentrations in our simulation differ quite a bit in values. And their units are not the same, I believe

Reply:
This merits some discussion.